build/rbac/rbac.yaml

```
@@ -74,6 +74,12 @@ metadata:
   labels:
     operator: rook
     storage-backend: ceph
+    app.kubernetes.io/name: rook-ceph
+    app.kubernetes.io/component: csi
+    app.kubernetes.io/part-of: rook-ceph
+    app.kubernetes.io/managed-by: Helm
+    app.kubernetes.io/created-by: rook-ceph-operator
```

These resources are both created and managed by Helm, this seems more accurate
```suggestion
    app.kubernetes.io/created-by: Helm
```